src/applications/resources-and-support/components/ResourcesAndSupportSearchApp.jsx

```diff
@@ -124,11 +136,10 @@ export default function ResourcesAndSupportSearchApp() {
             </>
           )}
 
-          {!errorMessage &&
-            !articles && (
-              <LoadingIndicator message="Please wait while we load the application for you." />
-            )}
-        </div>
+        {!errorMessage &&
+          !articles && (
+            <LoadingIndicator message="Please wait while we load the application for you." />
+          )}
```

Comment:
Instead of adding an `isSearchPage` I just wondering if we could create wrap this in the widget startup...

For instances -

```jsx
// applications/resources-and-support/widgets/StandaloneSearchForm.jsx

export default function StandaloneSearchFormWidget() {
  const [input, setInput] = useState('');
  return (
    <SearchBar onSearch={alwaysSubmit()}   userInput={input} onInputChange={setInput}/>
  )
}
```

Then we could use that from the widget startup instead of the app - `src/applications/static-pages/resources-and-support-search/index.js`. That we can remove that floating `isSearchPage` condition and the dependency on ` window.location.pathname`

Reply:
I need the wrapping content you have too I think though in `ResourcesAndSupportSearchApp`

Reply:
There's definitely a way to rework this though so that we're not dependent on `window.location.pathname` 🙂

Reply:
NVM definitely went with your idea! 😂

Reply:
With the widget separated out, do we still need any of the changes in this file?